src/addressResolvers/lens.ts

```diff
@@ -28,15 +28,23 @@ async function apiCall(filterName: string, filters: string[]) {
   return items;
 }
 
+function normalizeAddresses(addresses: Address[]): Address[] {
+  return addresses.filter(isEvmAddress);
+}
+
 function normalizeHandles(handles: Handle[]): Handle[] {
-  return handles.map(h =>
-    /^[a-z0-9-_]{5,31}\.lens$/.test(h) ? `lens/${h.replace(/\.lens$/, '')}` : ''
-  );
+  return handles
+    .map(h => (/^[a-z0-9-_]{5,31}\.lens$/.test(h) ? `lens/${h.replace(/\.lens$/, '')}` : ''))
+    .filter(h => h);
 }
 
 export async function lookupAddresses(addresses: Address[]): Promise<Record<Address, Handle>> {
+  const normalizedAddresses = normalizeAddresses(addresses);
```

Comment:
I feel like we can just update `addresses` variable, so that we don't have to update many things. but yea we can keep it this way if needed

Reply:
Updating incoming argument is never a good idea, as array as passed by reference, thus it's also updating it outside this function